src/handlers.js

```diff
@@ -1,11 +1,14 @@
+const securityModOrBroadcaster = (context) => {
+  const isBroadcaster = context["badges-raw"] && context["badges-raw"].startsWith("broadcaster");
+  return context.mod || isBroadcaster;
+};
+
 // =======================================
 // Command: !alert <text>
 // Description: will display whatever text comes after the !alert command
 // =======================================
 actionHandlers['!alert'] = {
-    security: (context, textContent) => {
-        return context.mod || (context["badges-raw"] != null && context["badges-raw"].startsWith("broadcaster"))
```

Comment:
`context["badges-raw"] != null` is the same as `context["badges-raw"]`

Reply:
In this instance it should be ok, but you should be careful when using an if in that way. You get potentially unexpected behaviour if the value is 0.